app/admin/rules/template.hbs

```diff
@@ -13,4 +13,5 @@
   </span>
 
   {{outlet}}
+  {{ember-notify messageStyle='bootstrap'}}
```

Comment:
I think we should put this in the application template, that way it's available anywhere.